src/userSurvey.js

```diff
@@ -122,9 +129,30 @@ function setPopupToNoPromptPage() {
     });
 }
 
+/**
+ * Initialize storage for the module.
+ * @private
+ */
+function initializeStorage() {
+    if (storageSpace === null) {
+        storageSpace = storage.createKeyValueStorage("webScience.userSurvey");
+    }
+}
+
+/**
+ * Called when the current survey is completed. Sets surveyCompleted to true in storage
+ * and changes the browser action popup to the survey's no prompt page.
+ * @private
+ */
+function completeSurvey() {
+    storageSpace.set("surveyCompleted", true);
+    setPopupToNoPromptPage();
+}
+
 /**
  * Prompt the user to respond to a survey.
```

Comment:
Would give a short description of what the function does (i.e., how it behaves in various circumstances), an example of how to use it when you're running just one survey in your study, and an example of how to use it when you're running two surveys.